scio-core/src/main/scala/com/spotify/scio/io/Taps.scala

```diff
@@ -70,9 +73,9 @@ trait Taps {
   private def tableExists(table: TableReference): Boolean =
     Try(BigQueryClient.defaultInstance().getTableSchema(table)).isSuccess
 
-  private def bigQueryTap(sqlQuery: String): BigQueryTap = {
+  private def bigQueryTap(sqlQuery: String, flattenResults: Boolean): BigQueryTap = {
```

Comment:
Not sure about exposing `JobConfigurationQuery` since all other options must be set in a specific way in our case.